seqr/model_utils.py

```diff
@@ -210,6 +213,9 @@ def convert_seqr_kwargs_to_xbrowse_kwargs(seqr_model, **kwargs):
     if seqr_class_name == "Individual" and "family" in xbrowse_kwargs:
         xbrowse_kwargs["project"] = getattr(seqr_model, "family").project
 
+    if seqr_class_name == "LocusList" and 'name' in xbrowse_kwargs:
+        xbrowse_kwargs['slug'] = xbrowse_kwargs['name'].replace(' ', '-').lower()
```

Comment:
might be better to call `slugify` method instead. (Other places in the code use it)

Reply:
or django.utils.text.slugify